src/rust/src/zip339_ffi.rs

```diff
@@ -63,7 +63,7 @@ pub extern "C" fn zip339_free_phrase(phrase: *const c_char) {
     if !phrase.is_null() {
         unsafe {
             // It is correct to cast away const here; the memory is not actually immutable.
-            CString::from_raw(phrase as *mut c_char);
+            drop(CString::from_raw(phrase as *mut c_char));
```

Comment:
This conflicts with #5459.

Reply:
This change should not have been included in this PR, nor in #5459, given that we'd already fixed it in the wallet integration branch. I dislike the history noise it creates. But oh well.